themes/src/main/resources/theme/emeritus-theme/login/template.ftl

```diff
@@ -0,0 +1,166 @@
+<#macro registrationLayout bodyClass="" displayInfo=false displayMessage=true displayRequiredFields=false>
```

Comment:
did we copy these code from the default theme and modified to our needs?

Reply:
yes